.travis.yml

```diff
@@ -34,5 +34,5 @@ script:
  - flake8 .
  - ./.gofmt.sh
  - ./.buildifier.sh
- - python -m unittest discover -s reconciletags -p 'reconciletags_test.py'
+ - bazel test reconciletags:reconciletags_test
```

Comment:
Could you put this after the bazel clean && build line? It should be a bit faster :)